### ISSUE TITLE
After editing profile information, the number of credentials published on the “homepage” and “Credentials” page has not changed, but the published status of a credentials detail page has been changed to unpublished.

### ISSUE THREAD

**Issue by xuliting** [elastos/Elastos.Trinity.DApps.DID#105]

**Comment by dchagastelles** [comment]
Yes, thats true. It's an easy fix. The thing is we are using two different ways to say a credentials is published or not, one of then just checks if there's an published credential with the same id as the local credential and the other check if the value of the credential changed.

Both ways have pros and cons. I think we should check the value also, but in this case, it can be misleading for the user because a published credential will "disappear" in the home page. The user had 5/10 credentials published he edit his name now he has 4/10. We can go this way and maybe alert after when there's a modified credential.

@kpachhai should we go this way?

**Comment by kpachhai** [comment]
yea the more we can improve the UI to make this less confusing, the better it is. I also think it may be better to alert the user whenever this happens. Any thoughts on this @BenjaminPiette ?

**Comment by xuliting** [comment]
unfixed
